rubix-spi/src/main/java/com/qubole/rubix/spi/ClusterManager.java

```diff
@@ -31,53 +29,26 @@
  */
 public abstract class ClusterManager
 {
-  private long splitSize;
-  private int nodeRefreshTime;
-
-  public abstract ClusterType getClusterType();
-
-  public void initialize(Configuration conf)
+  private static Log log = LogFactory.getLog(ClusterManager.class.getName());
 
-  {
-    splitSize = CacheConfig.getCacheFileSplitSize(conf);
-    nodeRefreshTime = CacheConfig.getClusterNodeRefreshTime(conf);
-  }
+  private int nodeRefreshTime;
 
-  public int getNodeIndex(int numNodes, String key)
+  public ClusterType getClusterType()
   {
-    HashFunction hf = Hashing.md5();
-    HashCode hc = hf.hashString(key, Charsets.UTF_8);
-    int initialNodeIndex = Hashing.consistentHash(hc, numNodes);
-    int finalNodeIndex = initialNodeIndex;
-    if (hc.asInt() % 2 == 0) {
-      finalNodeIndex = getNextRunningNodeIndex(initialNodeIndex);
-    }
-    else {
-      finalNodeIndex = getPreviousRunningNodeIndex(initialNodeIndex);
-    }
-
-    return finalNodeIndex;
+    return null;
   }
 
-  // This is the size in which the file will be logically divided into splits
-  public long getSplitSize()
+  public void initialize(Configuration conf)
   {
-    return splitSize;
+    nodeRefreshTime = CacheConfig.getClusterNodeRefreshTime(conf);
   }
 
   public int getNodeRefreshTime()
   {
     return nodeRefreshTime;
   }
 
-  public abstract boolean isMaster()
-      throws ExecutionException;
-
   // Nodes format as per the note above
   // Should return sorted list
```

Comment:
What note does this comment refer to? Seems like this could use rephrasing or removal